Source/Core/DiscIO/WbfsBlob.cpp

```diff
@@ -40,10 +40,8 @@ WbfsFileReader::WbfsFileReader(const std::string& filename)
 
 WbfsFileReader::~WbfsFileReader()
 {
-	for (u32 i = 0; i != m_files.size(); ++ i)
-	{
-		delete m_files[i];
-	}
+	for (file_entry* entry : m_files)
+		delete f;
```

Comment:
you have to change the `f` to `entry` otherwise you're trying to delete a non-existing variable.